Framework/Nexus/inc/MantidNexus/NexusIOHelper.h

```diff
@@ -46,7 +46,7 @@ namespace {
   case ::NeXus::UINT64:                                                        \
     return func_name<T, uint64_t>(__VA_ARGS__);                                \
   default:                                                                     \
-    throw std::runtime_error("Unknown type in Nexus file");                    \
+    throw std::runtime_error("NeXusIOHelper: Unknown type in Nexus file");     \
```

Comment:
can you provide more information for ? What group, what type?

Reply:
That's a little difficult to do. Converting the nexus types to strings give very little information and are not human readable.
I also don't know which one of the `...` arguments contains the file handle, as the order differs if there is an output buffer supplied or not.